internal/provider/auth_cert_test.go

```diff
@@ -207,13 +210,34 @@ func TestAuthLoginCert_Login(t *testing.T) {
 		w.Write(m)
 	}
 
+	tempDir := t.TempDir()
```

Comment:
Should we also update the handler func and `want` with the `auth_login` data as done for the other types?

Reply:
Not necessarily required.